### PR TITLE
chore(md-enhance): add tips to Normal Code Demo doc

### PR DESCRIPTION
**Pull request by GavinXue**
When we use unocss or tailwind in Code Demo, we may need import external css file. This pr add tips for someone who has these  requirements too.

### CODE REVIEW

File: docs/md-enhance/src/guide/demo/normal.md [hunk 1]
```diff
@@ -41,6 +41,32 @@ icon: code
 
 :::
 
+::: tip import external style
```
Comment (Mister-Hope) [inline docs/md-enhance/src/guide/demo/normal.md:44]:
```suggestion
::: tip Import external styles
```

File: docs/md-enhance/src/guide/demo/normal.md [hunk 2]
```diff
@@ -41,6 +41,32 @@ icon: code
...
+
+- If you need import external style (e.g., unocss generated css file), You can add `link` tag to import it. Setting in you code demo's html or js block.
```
Comment (Mister-Hope) [inline docs/md-enhance/src/guide/demo/normal.md:46]:
```suggestion
- If you need import to external styles (e.g., css files generated by unocss), you can add `link` tags using html or js blocks. E.g.
```

File: docs/md-enhance/src/guide/demo/normal.md [hunk 3]
```diff
@@ -41,6 +41,32 @@ icon: code
...
+```html
+<!-- change to your own href -->
+<link rel="stylesheet" href="__uno.css" />
+```
+
+@tab js example
+
+```js
+// add this to js code block, change to your own href
+const linkElem = window.document.createElement("link");
```
Comment (Mister-Hope) [inline docs/md-enhance/src/guide/demo/normal.md:61]:
```suggestion
const linkElem = window.document.createElement("link");

```

File: docs/md-enhance/src/zh/guide/demo/normal.md [hunk 4]
```diff
@@ -41,6 +41,32 @@ icon: code
 
 :::
 
+::: tip 引入外部样式
+
+- 如果需要演示中引入其他样式，比如 unocss 生成的样式，可以使用添加 `link` 的方式引入，在 html 或 js 中引入均可，如下示例：
```
Comment (Mister-Hope) [inline docs/md-enhance/src/zh/guide/demo/normal.md:46]:
```suggestion
- 如果需要在演示中引入其他样式 (如 unocss 生成的样式)，可以通过在 html 或 js 中添加 `link` 标签的方式引入。例子：
```

File: docs/md-enhance/src/guide/demo/normal.md [hunk 5]
```diff
@@ -41,6 +41,32 @@ icon: code
...
+linkElem.setAttribute("rel", "stylesheet");
+linkElem.setAttribute("href", "__uno.css");
+document.appendChild(linkElem);
+```
+
+:::
+:::
```
Comment (Mister-Hope) [inline docs/md-enhance/src/guide/demo/normal.md:68]:
```suggestion
:::
::::
```

Also the `::: tip` should be `:::: tip`.

Comment (Mister-Hope) [reply docs/md-enhance/src/guide/demo/normal.md]: You are not editing the rest, the outer nesting  container should increase itself to be `::::` `:::::` ....

Comment (GavinXue) [reply docs/md-enhance/src/guide/demo/normal.md]: sorry, I missed it. I will fix it ASAP

**Pull request merged**